### ISSUE TITLE
Add sample_weight computation for multi-label

### ISSUE THREAD

**Issue by muellerdo** [frankkramer-lab/aucmedi#16]
For each class: c
-> Compute n_samples / (n_classes * n_occurences_for_c)

compute sample weights by multiplying weights together for each sample

Check out this:
https://scikit-learn.org/stable/modules/generated/sklearn.utils.class_weight.compute_sample_weight.html
